packages/notification-spa/src/app/header/header.component.html

```diff
@@ -0,0 +1,10 @@
+<div class="header">
+    <div class="header__links">
+        <a [routerLink]="'/'" class="header__link pf-c-breadcrumb__link">Home</a> 
+        <em class="fa fa-angle-right"></em> 
+        <a [routerLink]="'/form'" class="header__link active pf-c-breadcrumb__link">Notification</a>
```

Comment:
Just noticed, change the routerLink url.

Also, Why does the label "Notification" from the breadcrumb redirect to the form? Shouldn't it redirect to the Dashboard/Index page? And the form should have an additional link after this called "Setup Notifications"

Reply:
Also, the side bar navigation links seem to be broken in the screenshot. The active page is not consistently highlighted in the screenshots. Test that and fix if required.

Reply:
@deshmukhmayur The screenshots seem very old. I've updated them. Also tested the sidebar seems to be working correctly.

Reply:
@deshmukhmayur Changed the breadcrumb name and the URL link

Reply:
@hybridx It still needs a link to the dashboard. i.e. It should be `Home > Notifications > Setup Notifications`.
Because once a user is in Setup Notifications form, they're still inside Notifications SPA, so it should have a link to go back to the Notifications SPA home i.e. Dashboard.

Reply:
If I understand it correctly, he home will actually point to one-platform home URL which we will add later, right? @deshmukhmayur

Reply:
Because '/' will only take it to the SPA home page.

Reply:
So, for now, I'll let home point to '/' and add a comment which to change it later

Reply:
@deshmukhmayur Added setup notifications